src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package Duke;
+import java.util.Scanner;
+
+
+/**
+ * Duke is the main class that will save and run the program
+ *
+ * @author Fang Yiye
+ */
+public class Duke {
+    private TaskList taskList;
+    private static Ui ui;
+    private Storage storage;
+    private String filePath;
+    private static Scanner sc;
+
+
+    /**
+     * Constructor of Duke to initialise ui, storage and scanner
+     *
+     */
+    public Duke() {
+        ui = new Ui();
+        this.sc = new Scanner(System.in);
+        this.taskList = new TaskList();
+        filePath = "/Users/yiye/Desktop/cs2103Projects/ip/Data/duke.txt";
```

Comment:
Would it better to declare the file path as a static constant in the Storage class? I think that this makes sense because Storage is supposed to handle all the file reading and writing operations, so the Duke class does not need to know the file path.